infrasim/init.py

```diff
@@ -25,12 +25,15 @@ def create_infrasim_directories():
     if os.path.exists(config.infrasim_home):
         shutil.rmtree(config.infrasim_home)
     os.mkdir(config.infrasim_home)
+    os.mkdir(config.infrasim_node_config_map)
```

Comment:
@xiar map folder is under infrasim_home directory, because infrasim_home is checked and removed in advance, this sub-folder doesn't need to be checked again.